demo/main.tscn

```diff
@@ -62,12 +62,12 @@ tracker = &"left_hand"
 pose = &"grip"
 
 [node name="LeftHandMesh" type="MeshInstance3D" parent="XROrigin3D/LeftHand"]
+visible = false
 mesh = SubResource("BoxMesh_3kt6b")
 
-[node name="LeftControllerFbRenderModel" type="OpenXRFbRenderModel" parent="XROrigin3D/LeftHand"]
```

Comment:
If I'm reading right, the controller render model is replaced by the hand tracking mesh. Should we have logic to instead switch between the two based on whether the user is using a controller, or using hand tracking?

Reply:
Hand tracking data is available even when using a controller, which would just result in the hand being shown on the controller, which is what some apps want. I think how to make controllers and hands interact should be up to the app, and in the case of the demo app here, I think it'd be cool to show both the controller and the hand at the same time. :-)

Reply:
If we are able to animate the hand model so that it conforms to the controller, then yes that would be great to have. Otherwise I was thinking more about checking the source for the hand tracking data and showing the controller render models if the source is from controllers, and the hand mesh instead if the source is from actual hand tracking.

This will especially help illustrate the new Quest multimodal capability of being able to use both controller and hands at the same time.

https://www.uploadvr.com/quest-3-pro-hands-controllers-simultaneous-multimodal/

Reply:
> If we are able to animate the hand model so that it conforms to the controller, then yes that would be great to have.

Yep, we can! When testing this while having controllers in my hands, the hands will be animated based on where I'm touching the controllers (you just can't see the controllers, because the controller models are gone). So, I think just keeping the controller models in there should allow this to work.

Reply:
Just tested and it looks great!

Although it looks weird when you're just using hand tracking and the controller render models remain attached to your hand.

I think when using controllers, we should show both controller render models and the hand meshes, but when only using hand tracking, we should hide the controller render models. We should be able to do that by checking the source of the hand tracking data.

Reply:
Added some logic in the latest push that utilizes `OpenXRInterface.get_hand_tracking_soruce()` to hide the controller models when hand tracking is being used vs. controllers. It would be handy to have a signal for when the sources change, maybe that's something we want to consider adding?

Reply:
Sounds like a good idea. Does the extension provide an event for it, or would we have to poll?

Reply:
There doesn't seem to be an event related to hand tracking data sources but the `XR_TYPE_EVENT_DATA_INTERACTION_PROFILE_CHANGED` event seems to occur every time the source changes. `XRPositionalTracker` has a [`profile_changed`](https://github.com/godotengine/godot/blob/39f90600bc553f2273e72708eb2a8d37a8517e86/servers/xr/xr_positional_tracker.cpp#L73) signal. Maybe it could be bound to [`XRController3D`](https://github.com/godotengine/godot/blob/39f90600bc553f2273e72708eb2a8d37a8517e86/scene/3d/xr_nodes.cpp#L473) along with the others?

Reply:
Yes, let's do that!